tests/test-check-fs.lua

```diff
@@ -59,7 +59,7 @@ require('tap')(function(test)
       assert(metrics['used']['t'] == 'int64')
       assert(metrics['avail']['t'] == 'int64')
       p(metrics['options'])
-      assert(metrics['options']['v']:find('hello_world'))
+      assert(metrics['options']['v']:find('hello_world') > 0)
```

Comment:
Why greater than zero?  `0` is truthy in lua and find returns `nil` if it's not found.